docs/io.md

```diff
@@ -0,0 +1,128 @@
+# Inputs and Outputs
+
+Auto-CORPus can be used directly on one file, or a list of files. However, to use it on
+a directory containing multiple files and nested directories, there is an assumed
+structure. This is useful for analysing an article along with its associated tables
+and supplementary information.
+
+## Input
+
+Auto-CORPus processes two categories of biomedical literature: **full text with tables**
+and **supplementary information**.
```

Comment:
I'm wondering if the "with tables" bit might be confusing. Could drop it and mention elsewhere that AC can process tables.